frontend/test/pytest/test_quantum_control.py

```diff
@@ -494,98 +524,921 @@ def use_legacy_opmath():
         yield cm
 
 
-@pytest.fixture(
-    params=[qml.operation.disable_new_opmath_cm, qml.operation.enable_new_opmath_cm],
-    scope="function",
-)
-def use_legacy_and_new_opmath(request):
-    with request.param() as cm:
-        yield cm
-
-
-class TestInitialization:
+class TestControlledInit:
     """Test the initialization process and standard properties."""
 
-    # pylint: disable=use-implicit-booleaness-not-comparison
+    temp_op = TempOperator("a")
+
     def test_nonparametric_ops(self):
-        """Test ctrl initialization for a non parameteric operation."""
-        base = qml.PauliX("a")
+        """Test pow initialization for a non parameteric operation."""
+
+        op = C_ctrl(
+            self.temp_op,
+            (0, 1),
+            control_values=[True, False],
+            work_wires="aux",
+        )
+
+        assert op.base is self.temp_op
+        assert op.hyperparameters["base"] is self.temp_op
+
+        # In C_ctrl, wires include the list of all wires
+        assert op.wires == Wires((0, 1, "a", "aux"))
+
+        assert op.control_wires == Wires((0, 1))
+        assert op.hyperparameters["control_wires"] == Wires((0, 1))
+
+        assert op.target_wires == Wires("a")
 
-        op = C_ctrl(base, control=["b"])
+        assert op.control_values == [True, False]
+        assert op.hyperparameters["control_values"] == [True, False]
 
-        assert op.base is base
-        assert op.hyperparameters["base"] is base
-        assert op.name == "C(PauliX)"
+        assert op.work_wires == Wires(("aux"))
+
+        assert op.name == "C(TempOperator)"
 
         assert op.num_params == 0
-        assert op.parameters == []
+        assert op.parameters == []  # pylint: disable=use-implicit-booleaness-not-comparison
```

Comment:
```suggestion
        assert not []
```

Reply:
I don't understand this suggestion. :D

Reply:
Sorry about it, I just connected it above.